packages/web/src/lib/core/auth/providers/internal/StitchRedirectCredentialImpl.ts

```diff
@@ -0,0 +1,17 @@
+import { AuthInfo, ProviderCapabilities } from "stitch-core";
+import StitchRedirectCredential from "../StitchRedirectCredential";
+
+export default class StitchRedirectCredentialImpl
```

Comment:
Since we're refactoring `StitchRedirectCredential` to not be a `StitchCredential`, maybe this should be `StitchInternalRedirectCredential implement StitchCredential` so it still works in `handleRedirect`